manifest_staging/charts/secrets-store-csi-driver/templates/_helpers.tpl

```diff
@@ -33,6 +33,10 @@ labels:
   helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
 {{- end -}}
 
+{{- define "sscd-psp.fullname" -}}
+{{- printf "%s-psp-mic" (include "sscd.name" .) -}}
```

Comment:
The name here shouldn't have `mic` suffix

Reply:
done